R/Arena.R

```diff
@@ -2577,14 +2577,17 @@ setMethod("findFeeding2", "Eval", function(object, time, mets, rm_own=T, ind_thr
 #' @param object An object of class Eval.
 #' @param time A numeric vector giving the simulation steps which should be plotted. 
 #' @param mets Character vector of substance names which should be considered
+#' @param print Should raw data be printed on the monitor?(Default:FALSE)             
 #' @return Graph (igraph)
 #' 
-setGeneric("findFeeding3", function(object, time, mets){standardGeneric("findFeeding3")})
+setGeneric("findFeeding3", function(object, time, mets,print=FALSE){standardGeneric("findFeeding3")})
 #' @export
 #' @rdname findFeeding3
-setMethod("findFeeding3", "Eval", function(object, time, mets){
+setMethod("findFeeding3", "Eval", function(object, time, mets,print=FALSE){
   mets = intersect(object@mediac,as.character(mets))
-  time = time+1
+  times<-time
```

Comment:
Should I change every "=" or should I change just mine "<-" ? That depends on you.....